epxresults/utils.py

```diff
@@ -570,3 +570,28 @@ def _read_list_variable_file(
     df = pd.DatFrame(np.array([float(x) for x in list_data[1:]]))
 
     return df
+
+
+def _is_valid_results_directory(path) -> bool:
+    """
+    Determine if `path` is consistent with being a FRED results directory.
```

Comment:
This is a handy utility

Reply:
I am a little worried that some other directory could meet those conditions.  I wish the FRED results directory was identified more robustly.

Reply:
Could you have methods that depend on specific FRED results directory structure throw a specific `InvalidFREDResultsDirectory` error if something isn't right? Seems like something that would be useful to log, but difficult to recover from. Also consistent with idiomatic [EAFP](https://docs.python.org/3/glossary.html#term-EAFP) approach.

Reply:
leaving this for a future PR.